examples/run_Hahn_echo.py

```diff
@@ -132,14 +133,10 @@
 )
 config_dict.write()
 # }}}
-subprocess.call(
-    [
-        "python",
-        "examples/proc_raw.py",
-        data.name(),
-        filename_out,
-        my_exp_type,
-    ],
-    cwd=r"C:\git\proc_scripts",
-    shell=True,
-)
+env = os.environ
+subprocess.call("which python",shell=True)
+subprocess.call((" ".join(["python",os.path.join(
+    os.path.split(os.path.split(pyspecProcScripts.__file__)[0])[0],
+    "examples", "proc_raw.py"),
+    data.name(),filename_out, my_exp_type])),
+    env=env)
```

Comment:
but we don't want to do this here, we want to do this inside your new function.  I think fix the issues with the new function first, and then do this there.